### PR TITLE
Add top-level core npm scripts defined for plugin generator

### LINKED ISSUE #3542: Update npm scripts to match the plugin generator
This makes it a lot easier for people who have already used plugins to develop against videojs.

### PR DESCRIPTION
**Pull request by misteroneill**
## Description
Fixes #3542 

A few of the core scripts were left out (the `build:*` ones) because they had no equivalent. They probably shouldn't be generator core scripts anyway.

## Specific Changes proposed
Add the following npm scripts: build, change, clean, docs, start.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,12 @@
   "homepage": "http://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
+    "build": "grunt dist",
+    "change": "grunt chg-add",
+    "clean": "grunt clean",
+    "docs": "grunt vjsdocs",
```
Comment (gkatsev) [inline package.json]: I'm not sure this is actually a thing, unfortunately.

**Comment by BrandonOCasey** [comment]
do we need prepublish here as well? otherwise LGTM

**Comment by gkatsev** [comment]
Nope, publishing happens separately right now.

**Comment by gkatsev** [comment]
LGTM

**Pull request closed without merging**